### ISSUE TITLE
False negative for supertype list with a trailing comment in indent rule

### ISSUE THREAD

**Issue by romtsn** [pinterest/ktlint#1205]
## Expected Behavior

Given the code below
```Kotlin
(31) public class TrailingCommaRule :
(32)   Rule("trailing-comma"),
(33)   Rule.Modifier.Last, // runs last to ensure that the linebreaks are already inserted by the indent and other rules
(34)   UsesEditorConfigProperties {
```
There should be no violations

## Observed Behavior

The trailing comment confuses the `indent` rule
```
/ktlint/ktlint-ruleset-experimental/src/main/kotlin/com/pinterest/ktlint/ruleset/experimental/trailingcomma/TrailingCommaRule.kt:33:24: Missing newline after "," (indent)
```

## Your Environment

* Version of ktlint used: latest `master`
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): [ktlint task](https://github.com/pinterest/ktlint/blob/29e3ef5382e6511e8728b794ac8558b050f54a23/build.gradle#L41)

**Comment by paul-dingemans** [comment]
Problem can not be reproduced.
